src/lib/puppeteer/screenshot/getInstagramScreenshot.ts

```diff
@@ -16,12 +16,18 @@ import { getErrorMessage } from "@/utils/errorUtils";
 import type { GetScreenshotOptions } from "@/app/try/route";
 
 import { extractPageMetadata } from "../core/extractPageMetadata";
-import { captureScreenshot } from "./captureScreenshot";
 
 interface FetchOgImageOptions {
 	logger: GetInstagramScreenshotOptions["logger"];
 	page: GetOrCreatePageReturnType;
 }
+const INSTAGRAM_VIEWPORT = {
+	deviceScaleFactor: 3,
+	hasTouch: true,
+	height: 844,
+	isMobile: true,
+	width: 390,
+};
```

Comment:
```suggestion
interface FetchOgImageOptions {
logger: GetInstagramScreenshotOptions["logger"];
page: GetOrCreatePageReturnType;
}

const INSTAGRAM_VIEWPORT = {
	deviceScaleFactor: 3,
	hasTouch: true,
	height: 844,
	isMobile: true,
	width: 390,
};
```

Leave proper line spacing